sources/packages/backend/apps/api/src/route-controllers/student/student.aest.controller.ts

```diff
@@ -116,7 +117,15 @@ export class StudentAESTController extends BaseController {
    * @param response file content.
    */
   @Get("files/:uniqueFileName")
-  @ApiNotFoundResponse({ description: "Requested file was not found." })
+  @ApiNotFoundResponse({
+    description:
+      "Requested file was not found or the user does not have access to it.",
+  })
+  @ApiForbiddenResponse({
+    description:
+      "Warning: This file has not been scanned and will be available to download once it is determined to be safe or " +
+      "Due to our security rules, the original file, [fileName], was deleted. Please re-check your file and attempt to re-upload.",
```

Comment:
Minor, Minor, the second sentence in line 127 can start lower case and I the "[fileName]" in the sentence would not be required IMO. Suggestion as below:
"due to our security rules, the original file was deleted."